shell/app/common/utils/str-num-date.tsx

```diff
@@ -14,8 +14,15 @@
 import i18n, { getCurrentLocale } from 'i18n';
 import { isArray } from 'lodash';
 import moment, { Moment } from 'moment';
+import 'moment/locale/zh-cn';
```

Comment:
is this import required?

Reply:
The internationalization of the moment here does not take effect without import this.

Reply:
This is imported in core, not take effect?

Reply:
Yes, this will only take effect if import it again here.